internal/github/checkStalePRs.go

```diff
@@ -70,28 +70,55 @@ func CheckStalePRs(ctx context.Context, githubClient *github.Client, cfg *config
 // Checks if a PR is stale based on the last update from team members
 func isStale(ctx context.Context, githubClient *github.Client, pr *github.PullRequest, teamMembers map[string]bool, cutoffDate time.Time) (bool, error) {
 	listOptions := &github.ListOptions{PerPage: 100}
+	if pr.GetCreatedAt().After(cutoffDate) {
+		slogs.Logr.Info("PR was created within the last seven days")
```

Comment:
This doesn't really tell us anything useful. When writing a logline, try to think of what the context might be where you'd actually care about the log's content. For this log, I'd only want to look at this if I were trying to figure out why a PR I thought should be flagged did not get flagged as stale. And so at a minimum, for this logline to be useful, it should tell us the repository and PR number. That way I can review instances of this logline and go "oh that's the PR I'm looking for, it wasn't flagged as stale because the API (or this program's logic) said it was created within the last 7 days.

Reply:
```suggestion
		slogs.Logr.Info("PR was created within the last seven days, so it cannot be stale", "PR", pr.GetNumber(), "repository", pr.Base.Repo.GetName())
```
Here's an example of how you would make this log useful.